Backend/src/main/webapp/card.jsp

```diff
@@ -84,25 +84,12 @@
         </thead>
         
         <tr>
-                <td>1</td>
-                <td>Grafikkarte</td>
-                <td>2</td>
-                <td>200€</td>
+                <td id="position"></td>
+                <td id="product"></td>
+                <td id="count"></td>
+                <td id="price"></td>
         </tr>
-        
-        <tr>
-                <td>1</td>
-                <td>Grafikkarte</td>
-                <td>2</td>
-                <td>200€</td>
-        </tr>
-        
-        <tr>
-                <td>1</td>
-                <td>Grafikkarte</td>
-                <td>2</td>
-                <td>200€</td>
-          </tr>
+
   </table>
 </div>    
     <br><br>
```

Comment:
checkout html form afterwards.. @line 100+... you should have only changed html stuff..